src/write_file.cpp

```diff
@@ -228,7 +228,12 @@ SEXP buildMatrixMixed(CharacterVector v,
         }else{
           // dt_str = as<std::string>(m(ri,i));
           dt_str = m(ri,i);
-          datetmp[ri] = Rcpp::Date(atoi(dt_str.substr(5,2).c_str()), atoi(dt_str.substr(8,2).c_str()), atoi(dt_str.substr(0,4).c_str()) );
+          try{
+            datetmp[ri] = Rcpp::Date(atoi(dt_str.substr(5,2).c_str()), atoi(dt_str.substr(8,2).c_str()), atoi(dt_str.substr(0,4).c_str()) );
+          }catch(...) {
+            std::cerr << "Error reading date:\n" << dt_str << "\nrow: " << ri+1 << "\ncol: " << i+1 << "\n";
```

Comment:
Hi @ChrisSUMich `std::cerr` should be `Rcpp::Rcerr` see https://gallery.rcpp.org/articles/using-rcout/

Reply:
I made the change and pushed. I only have a little experience with github pull requests, but I think the same PR is now updated with the newest version. including changing to Rcpp::Rcerr

Reply:
Thanks, all fine. FYI: prior to this, R CMD check looked liked this:

```txt
  File 'openxlsx/libs/x64/openxlsx.dll':

  Found '_ZSt4cerr', possibly from 'std::cerr' (C++)

    Object: 'write_file.o'
```